optimum/intel/openvino/modeling_base.py

```diff
@@ -264,6 +264,10 @@ def _from_transformers(
             "trust_remote_code": trust_remote_code,
         }
 
+        if model_id.startswith("timm/"):
+            model_kwargs["use_safetensors"] = False
```

Comment:
Why only for timm models ?

Reply:
@echarlaix, do you mean we should remove the check of "timm"?

Reply:
Yes not sure to understand why this is done for `timm` models, is this coming from an incompatibility between `transformers` `AutoClasses` and `timm` models ?

Reply:
Yes. Please see this issue I raised: https://github.com/huggingface/transformers/issues/25282

These are some of the incompatibility. There are some more.